team/index.html

```diff
@@ -274,29 +274,42 @@ <h1 id="team-members">Meet the Team</h1>
 
 <hr />
 
-<h2 style="line-height:3;">SSAGES Core Developers</h2>
+<h2 style="line-height:3;">Current SSAGES Development Team</h2>
```

Comment:
Maybe we can leave just one h2 header `SSAGES Development Team` and separate with subheaders or `<p>` block as here, active and past contributors, removing the `Past Contributors` header below?

Reply:
Done.